racs.go

```diff
@@ -100,6 +100,7 @@ type task struct {
 	kind  string
 	state string
 	time  string
+	cmd   *exec.Cmd
```

Comment:
I think we should use map[int]*exec.Cmd for activeTasks instead so we don't need to store cmd in each task.

Reply:
Fixed.